pyproject.toml

```diff
@@ -41,7 +40,7 @@ license = {text = "new BSD"}
 maintainers = [{name = "Bertrand Thirion", email = "bertrand.thirion@inria.fr"}]
 name = "nilearn"
 readme = "README.rst"
-requires-python = ">=3.7"
+requires-python = ">=3.8"
```

Comment:
I'm not sure we should already deprecate Python 3.8 ?

Reply:
I'm not sure either. 3.8 reaches EOL in October next year. We can discuss in the coredev meeting